docs/documentation/maya/skin.md

```diff
@@ -1,127 +1,185 @@
+    - *Tip*: It may be desirable to leave this map weight to 1 as a starting point and tweak some areas later on as the results of the skin simulation are seen.
 
-You can select the attribute to paint from the combo box exposed in the UI:
+    - *Tip*: Smooth the borders by using the smooth-Flood combination to make sure the simulation doesn’t find hard edges as this could create odd wrinkles (unless that’s something you are looking for!).
 
-<figure style="margin-left:30%;" markdown> 
-  ![Pain Tool Skin Attributes example](images/paint_tool_skin_attributes.png) 
-  <figcaption> Figure 3: Paintable attributes listed in the UI. </figcaption>
+- **Global Damping**: Set global damping per vertex in the simulated mesh. The greater the value per vertex the more it will attempt to retain its previous position.
+
+- **Max Sliding Multiplier**: Determines the size of the sliding area per vertex. It corresponds to the maximum distance to the closest point on the reference mesh computed on initialization. Greater values will allow for greater sliding but will have a greater computational cost.
+    
+    - *Tip*: For areas where sliding isn't required paint to 0. Use values closer to 1 in areas where more sliding freedom is desired.
+
+- **Mass**: Set individual mass values per vertex in the simulated mesh.
+
+<figure>
+  <img src="images/skin_paint_example.png" caption="AdonisFX Paint Tool"> 
+  <figcaption><b>Figure 1</b>: Example of painted weights on the skin of a bear character. From left to right: Hard Constraints, Slide Constraints and Soft Constraints.</figcaption>
 </figure>
 
+> [!NOTE]
+> - *Hard*, *Soft* and *Slide* values are normalized for each vertex. Make sure to paint the values that you want to give priority to at the end in order to avoid the internal normalization override them in further strokes.
+
 # Attributes
 
 [^1]: Soft range: higher values can be used.
 
 #### Solver Attributes
-| Attribute            | Type  | Value     | Range/Options     | Description                  |
-| :------------        | :---  | :----     | :------------     | :--------------------------- |
-| Iterations           | Long  | 3         | \[1, 10\] [^1]    | Number of iterations that the solver will execute per simulation step. |
-| Material             | Enum  | Leather   | <ul><li>Fat</li><li>Muscle</li><li>Rubber</li><li>Tendon</li><li>Leather</li><li>Wood</li><li>Concrete</li></ul> | Solver stiffness. The materials are listed from lowest to highest stiffness |
-| Stiffness Multiplier | Float | 1.0       | \[0.0, 2.0\] [^1] | Multiplier factor to scale up or down the material stiffness. |
+ - **Enable** (Boolean, True): Flag to enable or disable the deformer computation.
+ - **Iterations** (Integer, 3): Number of iterations that the solver will execute per simulation step. Greater values mean greater computational cost.
+     - Has a range of \[1, 10\] [^1]
```

Comment:
Let's get rid of footnotes to avoid forcing the reader to slide down and up.

Reply:
Alright! But should I replace it with something like this?

```suggestion
     - Has a soft range of \[1, 10\]
```

Reply:
`- Has a range of \[1, 10\]. Upper limit is soft, higher values can be used`

Reply:
Applied!